lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/interfaces.cpp

```diff
@@ -252,3 +241,99 @@ void M5Stack_UWB_Trncvr::transfer() {
     }
   }
 }
+
+long KillSwitchRelay::kill_time;
+bool KillSwitchRelay::dead;
+
+void KillSwitchRelay::init() { 
+  pinMode(kill_pin, OUTPUT); 
+  reset();
+  KillSwitchRelay::kill_time = millis();
+}
```

Comment:
KillSwitchRelay::dead doesn't seem to be initialized in init(). all vars should be initialized to avoid undefined behavior

Reply:
it's initialized in reset, but I can explicitly initialize it as well